components/03-modules/slider/_slider.scss

```diff
@@ -0,0 +1,178 @@
+$slider__margin                  : $spacer--medium !default;
+$slider__item-padding            : $spacer--medium !default;
+$slider__transition              : $transition-base !default;
+$slider__navigation-icon__spacing: 48px !default;
+$slider__name-font-weight        : $font-weight-base !default;
+$slider__name-font-size          : $font-size-base !default;
+
+$slider__price-font-weight       : $font-weight-bold !default;
+$slider__price-color             : $black !default;
+$slider__price-old-color         : $black !default;
+$slider__price-old-font-weight   : $font-weight-base !default;
+$slider__price-old-font-size     : $font-size-extra-small !default;
+$slider__price-new-color         : $color-primary !default;
+$slider__price-new-font-wieght   : $font-weight-bold !default;
+$slider__price-new-font-size     : $font-size-small !default;
+
+$slider__icon-spacing            : 25px !default;
+$slider__icon-fill               : $green !default;
+$slider__icon-fill-hover         : #007025 !default;
+
+.slider {
+    margin-bottom: $slider__margin;
+    position: relative;
+
+    &__title {
+        margin-bottom: $slider__margin * 2;
```

Comment:
I believe you should use $spacer-large for this one, what do you think , Filip ?

Reply:
yes, you are right again. An addition to this I deleted `$slider__margin` variable and put here just `$spacer...` variables

Reply:
ok, moved to specific variables